src/d3-parliament.js

```diff
@@ -0,0 +1,274 @@
+/* eslint-disable no-param-reassign */
+import * as d3 from 'd3';
```

Comment:
Not sure if this is worth doing, but I believe we can pull in d3 modules individually like lodash to reduce the overall size of this package bundle. [d3-dispatch](https://www.npmjs.com/package/d3-dispatch) and [d3-selection](https://www.npmjs.com/package/d3-selection) for example.

Reply:
Good idea.